verticapy/vdataframe.py

```diff
@@ -11706,21 +11719,22 @@ def to_db(
             db_filter = " AND ".join([f"({elem})" for elem in db_filter])
         db_filter = f" WHERE {db_filter}" if (db_filter) else ""
         if relation_type == "insert":
-            query = "INSERT INTO {}{} SELECT {}{} FROM {}{}{}".format(
+            query = "INSERT INTO {0}{1}{2} SELECT {3}{4} FROM {5}{6}{7}".format(
+
```

Comment:
why is there extra line space here?

I think 1 input is missing. There are 8 required and only 7 lines,